.circleci/config.yml

```diff
@@ -1,28 +1,36 @@
-release-tags-and-branches: &release-tags-and-branches
-  filters:
-    tags:
-      ignore:
-        - /^.*-SNAPSHOT/
-        - latest
-    branches:
-      only: /^release\/.*/
+aliases:
+  release-tags-and-branches: &release-tags-and-branches
+    filters:
+      tags:
+        ignore:
+          - /^.*-SNAPSHOT/
+          - latest
+      branches:
+        only: /^release\/.*/
 
-release-tags: &release-tags
-  filters:
-    tags:
-      ignore:
-        - /^.*-SNAPSHOT/
-        - latest
-        - /^.*-amazon*/
-    branches:
-      ignore: /.*/
+  release-tags: &release-tags
+    filters:
+      tags:
+        ignore:
+          - /^.*-SNAPSHOT/
+          - latest
+          - /^.*-amazon*/
+      branches:
+        ignore: /.*/
+  
+  release-branches: &release-branches
+    filters:
+      tags:
+        ignore: /.*/
+      branches:
+        only: /^release\/.*/
 
 version: 2.1
 orbs:
   android: circleci/android@0.2.1
   android-wordpress-orb: wordpress-mobile/android@1.0.15
   gcp-cli: circleci/gcp-cli@2.1.0
-  revenuecat: revenuecat/sdks-common-config@1.1.0
+  revenuecat: revenuecat/sdks-common-config@dev:01a27fcc557ae1c84721b437e6a7720a9b2e126d
```

Comment:
temporary until https://github.com/RevenueCat/sdks-circleci-orb/pull/5 is released